lib/cli.js

```diff
@@ -33,6 +33,9 @@ var processArgs = function(argv, options, fs, path) {
 
   if (helper.isString(options.logLevel)) {
     options.logLevel = constant['LOG_' + options.logLevel.toUpperCase()] || constant.LOG_DISABLE;
+  } else if (helper.isDefined(options.logLevel)) {
```

Comment:
yes.

I think it might be better to just show a warning when the user misconfigures `--log-level`. It won't affect Karma. This is different from failing to instantiate a preprocessor or launcher (we should fail in that case, because it might pass the tests - a browser that would fail didn't start).

What do you think?